packages/typespec-python/test/generated/authentication-api-key/authentication/apikey/_operations/_operations.py

```diff
@@ -39,17 +38,10 @@ def build_api_key_valid_request(**kwargs: Any) -> HttpRequest:
 
 
 def build_api_key_invalid_request(**kwargs: Any) -> HttpRequest:
-    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
```

Comment:
this is a good change, we shouldn't have sent accept headers for an operation that returns none